changelogs/unreleased/1638-add-soft-delete-option.yml

```diff
@@ -3,6 +3,6 @@ description: >
     set deletion if the model is exporting resources included in this set.
 issue-nr: 1638
 issue-repo: inmanta-lsm
-change-type: minor
-destination-branches: [master, iso7]
+change-type: patch
```

Comment:
Right, hadn't considered that it would have to be a patch to go in iso6. Still, it's more of a bugfix addition than a real feature.